src/dagre/position/bk.js

```diff
@@ -73,7 +71,7 @@ function findType1Conflicts(g, layering) {
     return layer;
   }
 
-  _.reduce(layering, visitLayer);
+  layering.reduce(visitLayer);
```

Comment:
I've tried testing this with the https://github.com/mermaid-js/mermaid project, and it looks like I have found an issue.

It looks like `_.reduce([], myFunc)` returns `undefined`.

However, `[].reduce(myFunc)` instead throws an error:

```
 FAIL  packages/mermaid/src/mermaidAPI.spec.ts > mermaidAPI > render > accessibility > stateDiagram-v2 > should set aria-roledescription to the diagram type AND should call addSVGa11yTitleDescription
TypeError: Reduce of empty array with no initial value
 ❯ findType1Conflicts node_modules/.pnpm/dagre-d3-es@file+..+dagre-es+dagre-d3-es-7.0.11.tgz/node_modules/dagre-d3-es/src/dagre/position/bk.js:74:12
 ❯ positionX node_modules/.pnpm/dagre-d3-es@file+..+dagre-es+dagre-d3-es-7.0.11.tgz/node_modules/dagre-d3-es/src/dagre/position/bk.js:354:33
 ❯ position node_modules/.pnpm/dagre-d3-es@file+..+dagre-es+dagre-d3-es-7.0.11.tgz/node_modules/dagre-d3-es/src/dagre/position/index.js:10:18
 ❯ node_modules/.pnpm/dagre-d3-es@file+..+dagre-es+dagre-d3-es-7.0.11.tgz/node_modules/dagre-d3-es/src/dagre/layout.js:42:30
 ❯ notime node_modules/.pnpm/dagre-d3-es@file+..+dagre-es+dagre-d3-es-7.0.11.tgz/node_modules/dagre-d3-es/src/dagre/util.js:250:10
 ❯ runLayout node_modules/.pnpm/dagre-d3-es@file+..+dagre-es+dagre-d3-es-7.0.11.tgz/node_modules/dagre-d3-es/src/dagre/layout.js:42:3
 ❯ node_modules/.pnpm/dagre-d3-es@file+..+dagre-es+dagre-d3-es-7.0.11.tgz/node_modules/dagre-d3-es/src/dagre/layout.js:19:31
 ❯ notime node_modules/.pnpm/dagre-d3-es@file+..+dagre-es+dagre-d3-es-7.0.11.tgz/node_modules/dagre-d3-es/src/dagre/util.js:250:10
 ❯ node_modules/.pnpm/dagre-d3-es@file+..+dagre-es+dagre-d3-es-7.0.11.tgz/node_modules/dagre-d3-es/src/dagre/layout.js:19:5
```

I think we could replace it with the following and it should be fine, though:

```suggestion
  layering.length > 0 && layering.reduce(visitLayer);
```